foundations/javascript_basics/arrays_and_loops.md

```diff
@@ -17,16 +17,16 @@ This section contains a general overview of topics that you will learn in this l
 
 Strings and numbers may be our building blocks, but as your scripts get more complex, you're going to need a way to deal with large quantities of them. Luckily, JavaScript has a couple of data types that are used for just that. An Array is an ordered collection of items (Strings, numbers, or other things).
 
-1. [This tutorial](https://www.w3schools.com/js/js_arrays.asp) is a great introduction.
-1. [This article](https://www.w3schools.com/js/js_array_methods.asp) covers some of the most useful built-in array methods. These fundamentals are something you'll use every day, so don't rush too much and miss out!
-1. [Web Dev Simplified video](https://www.youtube.com/watch?v=7W4pQQ20nJg) explains an overview of arrays in JavaScript in about 6 minutes.
+1. Go through [JavaScript Arrays from W3Schools](https://www.w3schools.com/js/js_arrays.asp).
+1. [JavaScript Array Methods](https://www.w3schools.com/js/js_array_methods.asp) covers some of the most useful built-in array methods. covers some of the most useful built-in array methods. These fundamentals are something you'll use every day, so don't rush too much and miss out!
+1. This [JavaScript Arrays crash course](https://www.youtube.com/watch?v=7W4pQQ20nJg) explains an overview explains an overview of arrays in JavaScript in about 6 minutes.
```

Comment:
```suggestion
1. This [JavaScript Arrays crash course](https://www.youtube.com/watch?v=7W4pQQ20nJg) explains an overview of arrays in JavaScript in about 6 minutes.
```

Reply:
@MaoShizhong I have corrected the typos, please check and let me know if everything is in order.